meeting-minutes/2024/2024-07-25-TOC-meeting-record.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: 2024-07-25 TOC Meeting Record
+parent: 2024
+grand_parent: Meeting Minutes
+nav_exclude: true
+---
+
+![Antitrust Policy Notice](../images/antitrust-policy-notice.png "Antitrust Policy Notice")
+![All are Welcome in the Hyperledger Community](../images/all-are-welcome.png "All are Welcome in the Hyperledger Community")
+
+Hyperledger is committed to creating a safe and welcoming community for all. For more information please visit our Code of Conduct: [Hyperledger Code of Conduct](https://toc.hyperledger.org/governing-documents/code-of-conduct.html)
+
+# Announcements
+- The [Hyperledger /dev/weekly developer newsletter](https://wiki.hyperledger.org/pages/viewpage.action?pageId=39618905) goes out each Friday to hundreds of Hyperledger developers. It is a collaborative effort. If you have a project release, pull request, community event, and/or relevant article you would like highlighted next week, please [leave a comment for consideration on the upcoming newsletter wiki page](https://wiki.hyperledger.org/display/DR/2024)
+- Maintainer Days / Member Summit 2024: Find out more here: [https://www.hyperledger.org/hyperledger-member-summit-2024](https://www.hyperledger.org/hyperledger-member-summit-2024)
+
+# Quarterly reports
+- [2024 Q2 Hyperledger Caliper](https://github.com/hyperledger/toc/pull/274)
+- [2024 Q3 Hyperledger Identus](https://github.com/hyperledger/toc/pull/271)
+- [2024 Q3 Hyperledger Web3j](https://github.com/hyperledger/toc/pull/275)
+- [2024 Q3 Hyperledger Cacti](https://github.com/hyperledger/toc/pull/277)
+- [2024 Q3 Hyperledger Fabric](https://github.com/hyperledger/toc/pull/278)
+- Please review any [outstanding quarterly reports](https://github.com/hyperledger/toc/pulls?q=is%3Apr+is%3Aopen+label%3Aquarterly-report+user-review-requested%3A%40me).
+
+# Upcoming reports
+- 2024 Q3 Hyperledger Aries (due August 1, 2024)
+- 2024 Q3 Hyperledger AnonCreds (due August 1, 2024)
+- 2024 Q3 Hyperledger Indy (due August 1, 2024)
+- [2024 TOC Project Update Calendar](../../project-reports/2024/2024-updates.md)
+
+# Discussion
+- [CommunityOne Bot](https://communityone.io/) - Fay
+- [Infrastructure changes for LF Decentralized Trust](https://docs.google.com/document/d/1E3cYIGIckXyI2h74Ee7_DCxi48PQmJ38rCnbnSZ9Clg/edit)
+- Task force: OpenSSF Scorecard
+
+# Recordings
+- [Recordings are available on the Hyperledger calendar](https://zoom-lfx.platform.linuxfoundation.org/meetings/hyp)
```

Comment:
@ryjones 
I've noticed this link is broken lately, it returns "Project Not Found". Where are the recordings stored now?

Reply:
That's new. You should be able to go to https://openprofile.dev/my-meetings to see any meetings that you attended.

Reply:
@denyeart @tkuhrt I've fixed the link. The slug change broke it.

Reply:
Thanks, works now!